app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -142,5 +144,13 @@ def create_cloud_tenant(options)
     def self.display_name(number = 1)
       n_('Cloud Manager', 'Cloud Managers', number)
     end
+
+    def destroy_mapped_tenants
+      if source_tenant
+        source_tenant.all_subtenants.destroy_all
+        source_tenant.all_subprojects.destroy_all
```

Comment:
Is it possible for these to be shared by any other providers?

Reply:
I don't believe so, the mappings should be unique to one provider as far as I understand it.